easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb

```diff
@@ -0,0 +1,27 @@
+name = 'MUMPS'
+version = '5.3.5'
+versionsuffix = '-parmetis'
```

Comment:
I know nothing about MUMPS, Metis and Parmetis, but I noticed that this is the first MUMPS EasyConfig that was build against Parmetis. Can you enlighten me a bit: why would one build with one or the other? It seems a bit surprising that no one build against Parmetis before